src/bin/pg_autoctl/pgsql.c

```diff
@@ -1310,6 +1320,41 @@ pgsql_get_wal_lag_from_standby(PGSQL *pgsql, int64_t *walLag)
 }
 
 
+/*
+ * pgsql_get_received_lsn_from_standby queries a standby PostgreSQL server to get the
+ * received_lsn value from the pg_stat_wal_receiver system view.
```

Comment:
Why choose the `received_lsn` value from the view? I think that's the main thing this comment should explain here. To be faire the next sentence does half of the job, explaining what the value is. It still does not explain why this is the one that we want to read.

Reply:
added some more explanation